test/Base_Tests/src/Network/Enso_Cloud/Enso_File_Spec.enso

```diff
@@ -41,6 +41,12 @@ add_specs suite_builder setup:Cloud_Tests_Setup = suite_builder.group "Enso Clou
         Data.list test_root.get . map .name . should_contain "test_file.json"
         Data.list test_root.get.path . map .name . should_contain "test_file.json"
 
+    group_builder.specify "list should sort its output" <|
+        r = Enso_File.home.list
+        r.should_be_a Vector
+        r . should_equal (r.sort on=.path)
+        r . should_equal (r.sort on=.enso_path)
```

Comment:
`enso_path` is private, I don't think the second check can work?

Reply:
Removed.